src/nft/components/bag/BagFooter.tsx

```diff
@@ -531,7 +549,14 @@ export const BagFooter = ({ totalEthPrice, fetchAssets, eventProperties }: BagFo
                 <ThemedText.SubHeaderSmall>
                   <Trans>Pay with</Trans>
                 </ThemedText.SubHeaderSmall>
-                <CurrencyInput onClick={() => (bagIsLocked ? undefined : setTokenSelectorOpen(true))}>
+                <CurrencyInput
+                  onClick={() => {
+                    if (bagIsLocked) return
```

Comment:
nit: could you do `if(!bagIsLocked) { doThings()... }` with no return?

Reply:
ok but hear me out 0 levels of indentation vs 1

Reply:
only if we change from spaces to tabs